memory/memhotplug.py

```diff
@@ -65,19 +62,19 @@ def get_hotpluggable_blocks(path, ratio):
         if memory.is_hot_pluggable(block):
             mem_blocks.append(block)
 
-    def f(num):
+    def percent(num):
```

Comment:
docstring please

Reply:
@harish-24

Reply:
@narasimhan-v Changes pushed. Forgot to put it here.

Reply:
@harish-24 this function is doing the reverse of percent.

Reply:
@narasimhan-v pushed with appropriate definition.

Reply:
@harish-24 Thanks, I am merging this. Maybe you can send a patch that optimises the function later.